### ISSUE TITLE
Requirement for '--no-sandbox' flag  when running the AppImage on Ubuntu 24.04.

### ISSUE THREAD

**Issue by meiphoo7-Mae** [ferdium/ferdium-app#1535]
### Avoid duplicates

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22bug+%3Abeetle%3A%22) for a bug report similar to mine, in vain

### Ferdium Version

6.7.1-nightly.5

### What Operating System are you using?

Ubuntu

### Operating System Version

24.04 (development version)

### What arch are you using?

x64

### Last Known Working Ferdium version

6.7.1-nightly.5 (for Ubuntu 23.xx)

### Expected Behavior

The Appimage is supposed to start normally as it did before.

### Actual Behavior

When using Ubuntu 24.04 running the AppImage fails with: [8291:0117/011731.447634:FATAL:credentials.cc(127)] Check failed: . : Permission denied (13)

### Steps to reproduce

Upgrade Ubuntu to version 24.04 (which is officially to be released in april 2024) and see the problem happening.

### Additional information

I'm an early adopter so I ran into this problem using Ubuntu 24.04 which is still under development. The problem can be mitigated by running the appimage with the  **--no-sandbox** flag. However when I inspect the running processes by issuing the command _ps ax_ I see this flag is now present twice for each Ferdium process so apparently the flag was already present in ferdium but it seems it is not in the right place to avoid the problem. The same problem, but with a different software package, is described [here](https://discourse.joplinapp.org/t/possible-future-requirement-for-no-sandbox-flag-for-ubuntu-23-10/32160).
The problem does not affect Ubuntu 23 because the feature of concern isn't turned on for that branch.

**Comment by mcmxcdev** [comment]
Thanks for letting us know of this by being an early adopter of Ubuntu 24.04!

I will be upgrading myself from Ubuntu 23.10 as soon as the stable release drops. If the issue still exists, then we can get it fixed properly.

**Comment by palandovalex** [comment]
the same problem, I checked several applications that used to work fine for me.

**Comment by pepper3k** [comment]
I can confirm the same thing for the latest build of electrum wallet appimageon(on arch linux x64), so it might not be specific to ferdium

**Comment by eXistenZNL** [comment]
Can confirm on Xubuntu 24.04, adding --no-sandbox makes it start correctly.

**Comment by michaelcozzolino** [comment]
the issue still exists

**Comment by kd4pba** [comment]
Still a problem for me July 23

**Comment by Amirul898** [comment]
Same problem

**Comment by lifecrane** [comment]
many appimages don't work without --no-sanbox switch.. looks like ubuntu did not want to fix this problem..

**Comment by meiphoo7-Mae** [comment]
Yes, Ubuntu did not want to fix this problem. To be precise: they deliberately introduced this so called feature to increase security so it's pretty obvious that Ubuntu won't revert this.
But still: I'm baffled that this issue, about one year later, still attracts comments.
Also to run the Ferdium appimage properly in Wayland one needs a few flags anyway.
